src/shogun/lib/SGMatrix.cpp

```diff
@@ -388,6 +388,14 @@ void SGMatrix<T>::create_diagonal_matrix(T* matrix, T* v,int32_t size)
 }
 
 template <class T>
+SGMatrix<T> SGMatrix<T>::get_submatrix(index_t col_start, index_t col_end) const
```

Comment:
(maybe even rename the existing "get_subvector" for consistency. But nevermind for now